docs/reference/troubleshooting/disk/fix-other-node-out-of-disk.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[fix-other-node-out-of-disk]]
+== Fix other role nodes out of disk
+
+Elasticsearch can use dedicated nodes to execute other functions apart from storing data or coordinating the cluster,
+for example machine learning. If one or more of these nodes are running out of space, you need to ensure that they have
+enough disk space to function. If the health disk indicator reports that a node that is not a master and does not
```

Comment:
I think we should link to the health disk indicator docs here

Reply:
Good point, I will add it.